diagrams/artifact-tenant-isolation.html

```
@@ -0,0 +1,288 @@
+<title>Tenant Isolation Enforcement</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)